### ISSUE TITLE
strage error

### ISSUE THREAD

**Issue by tehseensagar** [xp4xbox/Puffader#45]
error: [Errno 2] No such file or directory: 'libopenblas.UWVN3XTD2LSS7SFIFK6TIQ5
GONFDBJKU.gfortran-win32.dll'

No idea

Regards

**Comment by tehseensagar** [comment]
I've removed python completly and re-install it with its require libraries needed by Puffader.

It's working fine. Closed it.

**Comment by tehseensagar** [comment]
I've removed python completly and re-install it with its require libraries needed by Puffader.

Compile it with Pyinstaller it work great . But yet with py2exe still having issue and above error ocured.

Regards

**Comment by tehseensagar** [comment]
Issue is resolve , I was working an other stuff where I installed numpy , scapy as well. due to these packages above error occurred. I've changed the path and now every thing set back to normal. If any one else having this issue so kindly check your windows path at %appdata%  in Roaming folder python rename it for a while and recompile Puffader.py file with python setup.py . It will work for sure . 

Thanks
